src/server/schema/resolvers/types/Life.ts

```diff
@@ -0,0 +1,14 @@
+import { GraphQLLifeResolvers } from '../definitions';
+
+const LifeGraphQL: GraphQLLifeResolvers = {
+    id: root => root._id,
+    firstName: root => root.firstName,
+    lastName: root => root.lastName,
```

Comment:
If the field in GraphQL and document is the same and there's no need to resolve the value from other sources, you can omit part